docs/source/conf.py

```diff
@@ -165,5 +165,5 @@
 
 # -- Options sphinx-multiversion -------------------------------------------
 smv_tag_whitelist = r"^v\d+\.\d+.\d+$"  # Include tags like "tags/v2.5.0"
-smv_branch_whitelist = "master"
+smv_branch_whitelist = "main"
```

Comment:
This is really important so `sphinx-multiversion` can generate docs for the new branch name.